### PR TITLE
Adding documentation of cli DOTNET_NOLOGO env variable

### PR DESCRIPTION
**Pull request by sfoslund**
## Summary

Updating docs to reflect addition of new CLI environment variable: see dotnet/cli#13275
Set as WIP until change is checked in.

cc @KathleenDollard, @wli3

### CODE REVIEW

File: docs/core/tools/dotnet.md [hunk 1]
```diff
@@ -249,6 +249,10 @@ dotnet myapp.dll
 
   Specifies the location of the servicing index to use by the shared host when loading the runtime.
 
+- `DOTNET_NOLOGO`
+
+  Specified whether .NET Core welcome and telemetry messages are displayed on first run. Set to `true` to mute these messages (values `true`, `1`, or `yes` accepted) or set to `false` to allow (values `false`, `0`, or `no` accepted). If not set, the default is `false` and the messages will be displayed on first run. Note that this flag has no effect on telemetry (see `DOTNET_CLI_TELEMETRY_OPTOUT` for opting out of sending telemetry).
```
Comment (tdykstra) [inline docs/core/tools/dotnet.md]: Specified --> Specifies

```suggestion
  Specifies whether .NET Core welcome and telemetry messages are displayed on first run. Set to `true` to mute these messages (values `true`, `1`, or `yes` accepted) or set to `false` to allow (values `false`, `0`, or `no` accepted). If not set, the default is `false` and the messages will be displayed on first run. Note that this flag has no effect on telemetry (see `DOTNET_CLI_TELEMETRY_OPTOUT` for opting out of sending telemetry).
```

**Review by tdykstra — APPROVED** [review]
Looks great -- one typo.

File: docs/core/tools/telemetry.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ The .NET Core tools collect usage data in order to help us improve your experien
 Read more about .NET Core CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry
 ```
 
+To disable this message and the .NET Core welcome message, set the `DOTNET_NOLOGO` environment variable to true. Note that this variable has no effect on telemetry opt out.
```
Comment (khellang) [inline docs/core/tools/telemetry.md]:
```suggestion
To disable this message and the .NET Core welcome message, set the `DOTNET_NOLOGO` environment variable to `true`. Note that this variable has no effect on telemetry opt out.
```

**Review by KathleenDollard — APPROVED** [review]

**Comment by sfoslund** [comment]
@tdykstra this should be ready to merge

**Pull request merged**